src/layout/RootLayout.tsx

```
@@ -1,19 +1,7 @@
-import { useEffect } from 'react';
-
-import { Outlet, useLocation } from 'react-router-dom';
-
-import { ROUTES } from '@/routes/paths';
+import { Outlet } from 'react-router-dom';
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)